app/presenters/concerns/claim_csv_helpers.rb

```diff
@@ -0,0 +1,33 @@
+module ClaimCSVHelpers
+  extend ActiveSupport::Concern
+
+  COMPLETED_STATES = %w[rejected refused authorised part_authorised].freeze
+  SUBMITTED_STATES = %w[submitted redetermination awaiting_written_reasons].freeze
+
```

Comment:
also don't you need a `included do` block to mixin the methods? in any event

Reply:
👍 have added included do block to the  (although everything seemed to work without?)